api/Window.json

```diff
@@ -4176,7 +4176,8 @@
               "version_added": true
             },
             "edge": {
-              "version_added": null
+              "version_added": true,
+              "notes" : "<code>behaviour: smooth</code> does not work in Edge."
```

Comment:
I think this is already covered by [the CSS data](https://github.com/mdn/browser-compat-data/blob/ad2ef05077b866931bcff90b829777862b13f3bf/css/properties/scroll-behavior.json#L25), so this line can be removed